README.md

```diff
@@ -0,0 +1,34 @@
+# ClipBit
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+
+## Introduction
+
+[ClipBit](https://pypi.org/project/clipbit/1.0.0/) is a CLI tool developed to generate an abstractive text summary of youtube videos. All you need to provide is a link to a Youtube
+video of your choice and the program will give a text summary.
+
+## Installation
+ClipBit is available for installation via the pip command. To install ClipBit on your system, run the command `pip install clipbit`

```

Comment:
We can just say, `run <command>`. We can remove the first sentence in this line.

Reply:
Done